src/rgw/driver/rados/rgw_bucket.cc

```diff
@@ -1521,10 +1521,20 @@ static int bucket_stats(rgw::sal::Driver* driver,
   ::encode_json("explicit_placement", bucket->get_key().explicit_placement, formatter);
   formatter->dump_string("id", bucket->get_bucket_id());
   formatter->dump_string("marker", bucket->get_marker());
-  formatter->dump_stream("index_type") << bucket_info.layout.current_index.layout.type;
   formatter->dump_int("index_generation", bucket_info.layout.current_index.gen);
-  formatter->dump_int("num_shards",
-		      bucket_info.layout.current_index.layout.normal.num_shards);
```

Comment:
would prefer not to remove these fields in case users are parsing the output of `radosgw-admin bucket stats`

`min_num_shards` will already be dumped as part of `radosgw-admin bucket layout`, so i'm not sure we need to add a `current_layout` section here